examples/Makefile.am

```diff
@@ -25,3 +25,7 @@ str_LDADD = ../src/.libs/libcdada.a -lstdc++
 bbitmap_SOURCES = bbitmap.c
 bbitmap_CPPFLAGS = -I$(top_srcdir)/include/
 bbitmap_LDADD = ../src/.libs/libcdada.a -lstdc++
+
+stack_u32_SOURCES = stack_u32.c
+stack_u32_CPPFLAGS = -I$(top_srcdir)/include/ -D__CDADA_INTERNAL_INCLUDE
```

Comment:
I just realised `examples/` folder should NOT define `-D__CDADA_INTERNAL_INCLUDE`. Original 2 examples were having it.

Could you add a commit on top to remove them from all of them? Thanks

Reply:
Done.